cpprest.lua

```diff
@@ -22,6 +22,8 @@ project "cpprest"
   defines {
     "_NO_PPLXIMP", -- prevent building a dynamic library
     "_NO_ASYNCRTIMP",
+    "CPPREST_EXCLUDE_BROTLI",
+    "CPPREST_EXCLUDE_WEBSOCKETS",
```

Comment:
Are these defines used in cpprest header files? I'm wondering if we need all of runtimecore to build with these defines too.

Reply:
`CPPREST_EXCLUDE_BROTLI` is not.

`CPPREST_EXCLUDE_WEBSOCKETS` is here: https://github.com/Esri/cpprestsdk/blob/runtimecore/Release/include/cpprest/details/basic_types.h#L120 but it is not relevant as it consider the case where the directive is not defined which is not this case. It also used in other header that are either not exposed or we are not using like `Release\include\cpprest\ws_client.h`